controllers/component_dependency_update_controller.go

```diff
@@ -264,6 +266,16 @@ func (r *ComponentDependencyUpdateReconciler) handleCompletedBuild(ctx context.C
 		repo = image[0:index]
 		tag = image[index+1:]
 	}
+	// find any configurations for files to nudge in
+	nudgeFiles := []string{}
+	for _, p := range pipelineRun.Spec.Params {
```

Comment:
Thinking about it a bit is a parameter really the best place for this? Wouldn't an annotation make more senses?